observability/microservices.md

```diff
@@ -0,0 +1,53 @@
+# Observability in Microservices
+
+microservices is a very popular software architecture, where the application is arranged as a collection of loosely coupled services. Some of those services can be written in different languages by different teams.
+
+## Motivations
+
+We need to consider special cases when creating a microservice architecture from the perspective of observability. The main reason for that is because we want to capture the interaction between those microservices are going to have when making request between each other that can't be correlated.
```

Comment:
```suggestion
We need to consider special cases when creating a microservice architecture from the perspective of observability. We want to capture the interactions when making requests between those microservices and correlate them.
```

Reply:
> 

For technical documentation, a passive voice is generally recommended. I'm guilty of using this tone for documentation, one thing that I'm trying to correct for some time now. One suggested way to reword this sentence - "It is pertinent to consider special cases from the perspective of Observability when creating a microservice architecture."
It's minor, feel free to not use this sentence